src/table/cells/handle-key-press.js

```diff
@@ -28,10 +28,20 @@ export const arrowKeysNavigation = (evt, rowAndColumnCount, cellCoord, selState)
       nextRow > 0 && (!isSelectedTable || nextRow !== 1) && nextRow--;
       break;
     case 'ArrowRight':
-      nextCol < rowAndColumnCount.columnCount - 1 && !isSelectedTable && nextCol++;
+      if (nextCol < rowAndColumnCount.columnCount - 1 && !isSelectedTable) {
+        nextCol++;
+      } else if (!isSelectedTable && nextRow < rowAndColumnCount.rowCount - 1) {
```

Comment:
Ok being a bit picky here, but you are checking `!isSelectedTable` twice. Cleaner IMO to first have an if checking `!isSelectedTable`, wrapping the other two cases. Minor detail, it is fine as it is

Reply:
you mean if we added some code to check if `isSelectedTable` is going to be true then break from that case?

Reply:
Done, I wrapper the both cases with `if (!isSelectedTable) { ... } ` first

Reply:
yes, but I'd say check `!isSelectableTable` and do stuff in that case